common/culture/traditions/00_regional_traditions.txt

```diff
@@ -476,26 +502,39 @@ tradition_horse_lords = {
 
 	is_shown = {
 		OR = {
-			like_mongolic_group_trigger = yes
-			like_turkic_group_trigger = yes
+			# Warcraft
+			has_cultural_pillar = heritage_yaungolic
+			has_cultural_pillar = heritage_zaetaric
+			has_cultural_pillar = heritage_tharian
+			has_cultural_pillar = heritage_cenaric
+			has_cultural_pillar = heritage_orcish
+			has_cultural_pillar = heritage_agamagganic
```

Comment:
Not sure about the quilboars. They rarely seen on boars.

Reply:
Weird. I thought I read that they did use mounts in the rpg, but upon looking again I can't seem to find it. I will remove them from the list.